src/main/resources/db/migration/V1_1__predictor_scores.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS offender_predictor
+(
+    offender_predictor_id           serial          primary key,
+    offender_predictor_uuid         uuid            not null unique,
+    predictor_type                  varchar(100)    not null,
+    algorithm_version               varchar(10)     not null,
+    predictor_score                 varchar(10)     not null,
+    crn                             varchar(20)     not null,
+    predictor_trigger_source        varchar(200)    not null,
+    source_id                       varchar(200)    not null,
+    source_answers                  JSONB,
```

Comment:
this will store the answers passed to the service to generate the score. Not 100% sure if we really need this

Reply:
i think we won't pass the answers itself, just the dto i am creating at the moment with the actual answers but in a different format